src/shared/common/components/workstack.jsx

```diff
@@ -302,6 +301,13 @@ class WorkstackAllocate extends Component {
                         </td>;
                     }
                 }
+                if (row.data.DueDate) {
+                    if (new Date(row.data.DueDate) <= new Date()) {
+                        return <td key={row.uuid + column.dataValueKey} className='govuk-table__cell date-warning'>
+                            <span>{value}</span>
```

Comment:
Is the span required?

Reply:
The css fiile does it based on the class dateWarning with a span. This is more of a revert into MC so don't want to make wider changes than required. Also without looking we might apply default span css to TD's. Happy to create a ticket to look at this further in the future.